packages/next/build/babel/plugins/react-loadable-plugin.ts

```diff
@@ -151,10 +151,12 @@ export default function ({
                 t.binaryExpression(
                   '+',
                   t.stringLiteral(
-                    relativePath(
-                      state.file.opts.caller.pagesDir,
-                      state.file.opts.filename
-                    ) + ' -> '
+                    (state.file.opts.caller.pagesDir
+                      ? relativePath(
+                          state.file.opts.caller.pagesDir,
+                          state.file.opts.filename
+                        )
+                      : state.file.opts.filename) + ' -> '
```

Comment:
This is not enough. The key is also used on the other side.

Reply:
https://github.com/vercel/next.js/blob/1d643eb6e1cc57faaecf02e6c3c5e1bb463b2027/packages/next/build/webpack/plugins/react-loadable-plugin.ts#L108-L110

Reply:
These files aren't loaded client-side, they are only being tested in the jest environment so this allows the babel transpile to complete and the tests to run, tested against the provided failing repo https://github.com/ballpit/dynamic-10.2-bug

Reply:
We could only allow this if the `NODE_ENV` is set to test 🤔

Reply:
This same issue occurs when running next@10.2 with the latest storybook so I'm unsure if NODE_ENV will cover all the breaking cases :(